### ISSUE TITLE
Can't connect to custom deployed using Desktop or mobile apps

### ISSUE THREAD

**Issue by dinhtungdu** [RocketChat/Rocket.Chat#1736]
*I deployed another one using docker as comment bellow.*
Hi,
I deployed Rocket.Chat on a Digital Ocean VPS, Ubuntu 14.04, I'm using upstart to manage rocketchat. I followed [this tutorial](https://www.digitalocean.com/community/tutorials/how-to-deploy-a-meteor-js-application-on-ubuntu-14-04-with-nginx) and had successful use rocket.chat via web browser. But not for desktop and mobile app.
I used self signed ssl. VPS Ubuntu 512MB ram. Nginx.
After searching, I see some comments that say: RC desktop/mobile app won't work if the RC server deployed with port other than 3000. So I changed my RC on VPS to port 3000, but It's not work. What can I do now.
Here is my upstart config: http://pastebin.com/apLsGTUV
And my Nginx vhost: http://pastebin.com/gR1W2kTE
Thank you so much!

**Comment by dinhtungdu** [comment]
Update:
I followed the Docker guide in Wiki and I can connect in Desktop/Mobile app with: http://my-url.com:3000
Still unable to use https to connect.

**Comment by blabj** [comment]
I had same problem with self-signed cert.  I used CADroid to install the self-signed cert on Android and then it worked.  For the desktop client, on linux, I installed the cert in Chrome, and then the client worked.

I found it strange, since there is an option in the administration section, to allow self-signed certs to be valid urls.. yet the clients refuse to connect.  Perhaps this is a bug in the client code?
